src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-dynamic-feature-android-gradle-plugin.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Configure Android Dynamic Feature
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Install configure
+metaDescription: 'How to configure the New Relic Android Gradle Plugin For Android Dynamic Feature '
+freshnessValidatedDate: never
+---
+
+What is Android Dynamic Feature?
+
+Android dynamic features refer to a functionality introduced by the Android platform that allows developers to modularize their applications and deliver specific features on-demand.
+
+The New Relic Gradle plugin introduces the ability to customize plugin task behavior for Android dynamic feature modules during the build process. To implement this feature, create a plugin extension in the Gradle build files that applies the New Relic plugin to the dynamic feature module. You can then make use of various configuration options available for customization.
+
+## Configure Android Dynamic Feature [#android-dynamic-feature]
+
+<Callout variant="important">
+This feature is available for agent versions 7.3.0 and higher.
+</Callout>
+
+To enable the Android Gradle Plugin and Agent for Android Dynamic Feature Module: Add the following to your Feature Module's `build.gradle`:
+
+```gradle
+
+apply plugin: 'com.android.dynamic-feature'
+apply plugin: 'newrelic'
+
+android {
+
+    compileSdkVersion versions.compileSdk
+    defaultConfig {
+        minSdkVersion versions.minSdk
+        targetSdkVersion versions.targetSdk
+    }
+}
+
+dependencies {
+    implementation project(':app')
+    compileOnly 'com.newrelic.agent.android:android-agent:7.3.0'
```

Comment:
implementation here instead of compileOnly?

Reply:
this is on dynamic feature module, if they want to use new relic static methods and they have already  implementation on their project level.